examples/bgp_route_summarization/main.tf

```diff
@@ -14,19 +14,15 @@ provider "aci" {
   insecure = true
 }
 
-resource "aci_tenant" "tenentcheck" {
-  name       = "test"
-  annotation = "atag"
-  name_alias = "alias_tenant"
+resource "aci_tenant" "tf_tenant" {
+  name = "tf_tenant"
 }
 
-resource "aci_bgp_route_summarization" "example" {
-
-  tenant_dn  = aci_tenant.example.id
-  name       = "example"
-  annotation = "example"
-  attrmap    = "example"
-  ctrl       = "as-set"
-  name_alias = "example"
-
+resource "aci_bgp_route_summarization" "bgp_rt_summ_pol" {
+  tenant_dn             = aci_tenant.tf_tenant.id
+  name                  = "bgp_rt_summ_pol"
+  description           = "from terraform"
+  attrmap               = "sample attrmap"
+  ctrl                  = sort(["summary-only", "as-set"])                 # Use the sorted list to handle identical changes
```

Comment:
What does this mean ->Use the sorted list to handle identical changes?

Isn't TF able to differentiate between  ["summary-only", "as-set"]  and ["as-set", "summary-only"]?